WalletWasabi.Gui/ViewModels/StatusBarViewModel.cs

```diff
@@ -78,6 +82,12 @@ public StatusBarViewModel()
 
 		public ReactiveCommand<Unit, Unit> UpdateCommand { get; set; }
 
+		public Network Network
+		{
+			get => _network;
+			set => this.RaiseAndSetIfChanged(ref _network, value);
```

Comment:
Is that `set` required given that the value never changes?

Reply:
It not hurt. Let it be.

Reply:
It will hurt when you will read it in five months, ten months, whatever. When somebody else will look at it, he may think "There must be a reason why it is there. I won't touch it just to not break anything." 

That's my reasoning for writing only necessary code. It does not hurt by itself but it has snowballing effect and if you want to save your time down the road, these things add up.

Reply:
Ok, makes sense.